src/svg.rs

```diff
@@ -37,13 +39,44 @@ impl Default for Svg {
         Self {
             name: Default::default(),
             size: Default::default(),
-            view_box: ViewBox { x: 0., y: 0., w: 0., h: 0. },
+            view_box: ViewBox {
+                x: 0.,
+                y: 0.,
+                w: 0.,
+                h: 0.,
+            },
             paths: Default::default(),
-            mesh: Default::default() }
+            mesh: Default::default(),
+        }
     }
 }
 
 impl Svg {
+    /// Loads an SVG from bytes
+    pub fn from_bytes(bytes: &[u8], path: &Path) -> Result<Svg, FileSvgError> {
+        let mut opts = usvg::Options::default();
+        opts.fontdb.load_system_fonts();
+        opts.fontdb.load_fonts_dir("./assets");
```

Comment:
Hm...as a premature optimization 🙈 do you think it would be a good idea to change the function to something like this?
```rust
pub fn from_bytes<P: AsRef<Path>>(bytes: &[u8], path: P, fonts: Option<P>) -> Result<Svg, FileSvgError> {
    let mut opts = usvg::Options::default();
    opts.fontdb.load_system_fonts();
    opts.fontdb.load_fonts_dir(fonts.unwrap_or("./assets"));
    ...
}
```

Reply:
> Hm...as a premature optimization 🙈 do you think it would be a good idea to change the function to something like this?
> 
> ```rust
> pub fn from_bytes<P: AsRef<Path>>(bytes: &[u8], path: P, fonts: Option<P>) -> Result<Svg, FileSvgError> {
>     let mut opts = usvg::Options::default();
>     opts.fontdb.load_system_fonts();
>     opts.fontdb.load_fonts_dir(fonts.unwrap_or("./assets"));
>     ...
> }
> ```

Yeah I think that makes sense. 👍